GameOfLife/displaygameboardonthecommandline.cpp

```diff
@@ -5,13 +5,15 @@ DisplayGameBoardOnTheCommandLine::DisplayGameBoardOnTheCommandLine()
 {
 }
 
-void DisplayGameBoardOnTheCommandLine::displayGameBoardOnTheCommandLine(std::string stringConvertedFromBoard)
+void DisplayGameBoardOnTheCommandLine::displayGameBoardOnTheCommandLine(std::string stringConvertedFromBoard,int lengthBoardWidth)
 {
         std::cout<<stringConvertedFromBoard;
-        std::cout<<"---------------------------- ";
+        for(int i =0; i< lengthBoardWidth; i++){
+            std::cout<<"-";
+        }
 }
 
 void DisplayGameBoardOnTheCommandLine::displayCounterOnTheCommandLine(int counterOfIterations)
 {
-        std::cout<<counterOfIterations<<std::endl;
+        std::cout<<" "<<counterOfIterations<<std::endl;
```

Comment:
The requirement says that the counter should be displayed in format: 'Iteration count: XX', so e.g.
---------- Iteration count : 9